mlflow/tensorflow/__init__.py

```diff
@@ -782,6 +815,12 @@ def train(original, self, *args, **kwargs):
 
         result = original(self, *args, **kwargs)
 
+        if log_input_examples:
+            nonlocal input_example_slice
+            from mlflow.tensorflow._autolog import extract_data_from_tf_input_fn
+
+            input_example_slice = extract_data_from_tf_input_fn(kwargs.get("input_fn"))
```

Comment:
Looking at the implementation of this function, it seems to throw if the data type is unsupported. We should only throw when the example slice is fetched within `resolve_input_example_and_signature`. See my comment on https://github.com/mlflow/mlflow/pull/5510/files#r851523221. Can we update this based on my recommendation in https://github.com/mlflow/mlflow/pull/5510/files#r851523221?